packages/app-page-builder/src/editor/plugins/elementSettings/accordion/AccordionItemsList.tsx

```diff
@@ -0,0 +1,211 @@
+import React, { useCallback } from "react";
+import styled from "@emotion/styled";
+import { css } from "emotion";
+import { useRecoilValue } from "recoil";
+import { ButtonIcon, ButtonSecondary } from "@webiny/ui/Button";
+import { useConfirmationDialog } from "@webiny/app-admin/hooks/useConfirmationDialog";
+import { ReactComponent as AddIcon } from "@webiny/app-admin/assets/icons/add-18px.svg";
+import { activeElementAtom, elementWithChildrenByIdSelector } from "~/editor/recoil/modules";
+import Accordion from "~/editor/plugins/elementSettings/components/Accordion";
+import { moveInPlace, useSortableList } from "~/hooks/useSortableList";
+import { Collapsable } from "~/editor/plugins/toolbar/navigator/StyledComponents";
+import { useUpdateElement } from "~/editor/hooks/useUpdateElement";
+import { ReactComponent as DragIndicatorIcon } from "~/editor/plugins/toolbar/navigator/assets/drag_indicator_24px.svg";
+import { ReactComponent as DeleteIcon } from "~/editor/assets/icons/delete.svg";
+import { createElement } from "~/editor/helpers";
+import { PbEditorElement } from "~/types";
+
+const accordionStyles = css`
```

Comment:
If possible, let's not use `emotion` package. We'll be removing this in the near future.

We only want to rely on `@emotion/styled` and `@emotion/react` for styling.

Reply:
On a second thought, just so I don't dwell on this too much, let's ignore this one. But for future PRs, please have this in mind. Thanks!